modules/spring-mock-mvc/src/test/java/io/restassured/module/mockmvc/ContentTypeTest.java

```diff
@@ -101,10 +101,11 @@
     }
 
     @Test public void
-    doesnt_duplication_of_content_type_with_default_charset() {
+    doesnt_duplication_of_content_type() {
```

Comment:
I've removed duplicated test of adds_default_charset_to_content_type_by_default and added one more for testing my changes. Git is showing changeset in little bit strange way.